docs/overhetalgoritmekader/index.md

```diff
@@ -1,83 +1,34 @@
-Het doel is dat het Algoritmekader gebruikers op een praktische wijze helpt bij het uitvoeren van hun werkzaamheden. 
-Het interactief kunnen doorzoeken van de informatie is daar een voorbeeld van.  
-
-Eind 2024 wordt de tweede versie van het Algoritmekader opgeleverd. 
-Dan moeten de vereisten en de maatregelen zijn uitgewerkt voor zover dan bekend. 
-Hierna zal het Algoritmekader in 'beheer' worden genomen. Dat betekent dat doorlopend de laatste ontwikkelingen, in afstemming met de omgeving, worden toegevoegd zodat het Algoritmekader actueel en betrouwbaar blijft.  
-
-## Samenwerking met de omgeving
-Een belangrijk uitgangspunt van het Algoritmekader is dat het kader op een open en transparante manier wordt ontwikkeld. 
-Hierbij wordt gebruik gemaakt van verschillende communicatiekanalen, zoals een communityplatform [Pleio](https://algoritmes.pleio.nl/), GitHub, periodieke bijeenkomsten en (waar passend) werkgroepen. 
-Informatie over algoritmes en AI worden uitgewerkt. 
-Zodra deze informatie op 'niveau' is, wordt dit in Github geplaatst en daarmee 'gepubliceerd'. 
-Dit noemen we *releases*. 
-
-De motivatie om op deze ‘open source’ manier aan dit Algoritmekader te werken is enerzijds de omgeving van deze informatie te voorzien en anderzijds om de omgeving uit te nodigen om hier feedback op te geven. 
-Werken in Github is voor het team Algoritmekader nieuw en experimenteel. 
-Het biedt de mogelijkheid voor alle geïnteresseerden om hun kennis en zienswijze te delen. 
-Daarmee kan (nieuwe) kennis snel onderdeel worden gemaakt van het Algoritmekader. 
-Daardoor blijft het Algoritmekader zo actueel mogelijk en wordt de inhoud breed gedragen.
-Tegelijkertijd vraagt dit om een aangepaste werkwijze en is hier bepaalde expertise voor nodig. 
-Het begin is gemaakt en het team Algoritmekader is nog lerende om hier optimaal invulling aan te geven. 
-Voor nu betekent dit concreet dat het langer kan duren voordat wordt gereageerd op suggesties of toevoegingen. 
-Er wordt gewerkt aan het inrichten van goede processen om snel en kundig te reageren. 
+# Over het algoritmekader 
+Het algoritmekader is een hulpmiddel voor overheden die algoritmes of AI (artificiële intelligentie) gebruiken. Hierin zoek je de regels en adviezen op per situatie. De informatie in het Algoritmekader is nog niet betrouwbaar. Het is een testversie. De definitieve versie verschijnt eind 2024.
+
+## Verantwoord gebruik van algoritmes en AI
+In het Algoritmekader vind je informatie over verantwoord gebruik van algoritmes en AI. Bijvoorbeeld:
+- Wie is waarvoor verantwoordelijk?
+- Hoe voorkom je discriminatie (bias) in een algoritme?
+- Hoe verwerk je gegevens op een veilige manier?
+- Welke algoritmes controleer je op mensenrechtenschendingen?
```

Comment:
```suggestion
- Hoe voorkom je je mensenrechtenschendingen?
```

Reply:
gaat niet per se om welke algoritmes dit zijn, maar meer hoe je mensenrechtenschending kan voorkomen, en hoe je er rekening mee kan houden in de ontwikkeling

Reply:
```suggestion
- Hoe voorkom je mensenrechtenschendingen?
```